samples: wifi: thread_coex: Readme update (#17183)
nrfconnect/sdk-nrf: samples/wifi/thread_coex/README.rst

.. include:: /includes/wifi_credentials_static.txt

.. note::
   ``menuconfig`` can also be used to configure ``Wi-Fi credentials``.

     See :ref:`zephyr:menuconfig` in the Zephyr documentation for instructions on how to run ``menuconfig``.

> * Wi-Fi throughput test: Set the :kconfig:option:`CONFIG_NET_CONFIG_PEER_IPV4_ADDR` Kconfig option appropriately as per the Wi-Fi interface IP address of the test PC on which **iperf** is run.

Author: 1. Wi-Fi connection is done before running the Wi-Fi throughput test. So, these two are in this order in the doc. 
2. Configuring the credentials is part of the Wi-Fi connection and so mentioned immediately after the Wi-Fi connection, Alternative way of configuring the credentials (using menuconfig)  is mentioned as part of this.
So, connection, configuring credentials and running throughput are in the right sequence.

> * Wi-Fi throughput test: Set the :kconfig:option:`CONFIG_NET_CONFIG_PEER_IPV4_ADDR` Kconfig option appropriately as per the Wi-Fi interface IP address of the test PC on which **iperf** is run.

Reviewer: Ok, thanks!

      1050759502         /dev/ttyACM3    VCOM1
      $

> 	  In this example, ``1050779496`` is the device ID of the first nRF7002 DK and ``1050759502`` is the device ID of the second nRF7002 DK.

Reviewer 2: ```suggestion
	  In this example, ``1050779496`` is the device ID of the first nRF7002 DK and ``1050759502`` is the device ID of the other one.
```

	  While connecting to a particular board, use the ttyACMx corresponding to VCOM1.
	  In the example, use ttyACM1 to connect to the board with device ID ``1050779496``.
	  Similarly, use ttyACM3 to connect to the board with device ID ``1050759502``.

#. Run the following commands to connect to the desired devices:

	.. code-block:: console

	   $ minicom -D /dev/ttyACM1 -b 115200
	   $ minicom -D /dev/ttyACM3 -b 115200

Programming DKs
===============

> Program the nRF7002 DK by completing the following steps:

Reviewer 2: ```suggestion
Complete the following steps to program the nRF7002 DK:
```

1. |open_terminal_window_with_environment|
#. Navigate to the :file:`<ncs code>/nrf/samples/wifi/thread_coex/` folder.
#. Run the following command:

   .. code-block:: console

	  $ west flash --dev-id <device-id> --hex-file build/merged.hex

Test procedure
==============

> Tables given below provide the procedure to run Wi-Fi only, Bluetooth LE-only, and combined throughput for different Thread roles.

Reviewer 2: ```suggestion
The following tables provide the procedure to run Wi-Fi only, Bluetooth LE-only, and combined throughput for different Thread roles.
```

	|               |             | on client** is seen on the DUT nRF7002 DK's UART console window.        |
	+---------------+-------------+-------------------------------------------------------------------------+

> When the sample runs Wi-Fi UDP throughput in client mode, a peer device (test PC) runs UDP throughput in server mode using the following command:

Reviewer 2: Does the user need to run that command or does it just happen?
If the user needs to run it, then rather:
"... runs UDP throughput in server mode.
Use the following command:"

> 	.. code-block:: console
> 
> 	   $ iperf -s -i 1 -u

Reviewer 2: ```suggestion
.. code-block:: console

   $ iperf -s -i 1 -u
```

> When the sample is executed, the Wi-Fi UDP throughput operates with the Device Under Test (DUT) in client mode. To run the UDP throughput in server mode on the peer device (test PC), use the following command.

Reviewer: ```suggestion
When the sample is executed, the Wi-Fi UDP throughput operates with the Device Under Test (DUT) in client mode.
To run the UDP throughput in server mode on the peer device (test PC), use the following command.
```
